scripts/apply-fixes.js

```diff
@@ -0,0 +1,25 @@
+const glob = require("glob");
+const fs = require("fs");
+
+glob.sync("themes/default/content/registry/packages/**/_index.md").forEach(path => {
+
+    // https://github.com/pulumi/registry/issues/5095
+    // Finds and replaces an errant set of backticks in the TF docs.
+    if (path === "themes/default/content/registry/packages/rancher2/api-docs/nodetemplate/_index.md") {
+        const content = fs.readFileSync(path, "utf8");
+        const lines = content.split("\n");
+        let precedingLine, nextLine = undefined;
+
+        lines.forEach((line, i) => {
+            if (line.endsWith("implements `fixNodeTemplateID()` that will update tfstate with proper id.")) {
```

Comment:
hmm honestly don't really like this. My concerns are around if the upstream changes we could incorrectly delete the following line that may end up causing just as much breakage especially if the line that follows happens to include html code or something important.  I don't really think this line approach is a good idea for that reason. Can we do something like check to make sure we have complete pairs of backticks and only remove last set of backticks if it is unclosed similar to what I am doing in the other PR I opened, except here it will only be isolated to this one file.

Reply:
ahh I see you checked the next lines below

Reply:
I figured it'd be extremely unlikely to encounter a situation in which these two lines would appear next to each other and be legit, but I went ahead and added a quick check to make sure there's also an add number of fences, just in case.